plugins/search-react/api-report.md

```diff
@@ -82,6 +82,11 @@ export class MockSearchApi implements SearchApi {
   query(): Promise<SearchResultSet>;
 }
 
+// @public (undocumented)
+export const MultiselectFilter: (
```

Comment:
I think we want to call this `MultiSelectFilter` instead

Reply:
done